Springboot tests are failing

/kind bug

```
[FAILED] [361.101 seconds]
odo list with devfile devfile has missing metadata when projectType and language is missing when the component is pushed [BeforeEach] should show 'nodejs' for 'Type' in odo list
  [BeforeEach] /go/src/github.com/redhat-developer/odo/tests/integration/cmd_devfile_list_test.go:354
  [It] /go/src/github.com/redhat-developer/odo/tests/integration/cmd_devfile_list_test.go:363

[...]

[FAILED] Timed out after 360.001s.
  Expected
      <string>:   __
       /  \__     Developing using the "aname" Devfile
       \__/  \    Namespace: cmd-devfile-list-test363ext
       /  \__/    odo version: v3.9.0
       \__/
      
      ↪ Running on the cluster in Dev mode
       •  Waiting for Kubernetes resources  ...
       ⚠  Pod is Pending
       ✓  Pod is Running
       •  Syncing files into the container  ...
      
 ✓  Syncing files into the container [1s]
       •  Building your application in container (command: defaultbuild)  ...
       ✗  Building your application in container (command: defaultbuild) [2s]
      Error occurred on Push - watch command was unable to push component: unable to exec command [/bin/sh -c cd /projects && (mvn clean -Dmaven.repo.local=/data/cache/.m2/repository package -Dmaven.test.skip=true) 1>>/proc/1/fd/1 2>>/proc/1/fd/2]: error while streaming command: command terminated with exit code 1
      
       •  Syncing files into the container  ...
      
 ✓  Syncing files into the container [1s]
       •  Building your application in container (command: defaultbuild)  ...
       ✗  Building your application in container (command: defaultbuild) [2s]
      Error occurred on Push - watch command was unable to push component: unable to exec command [/bin/sh -c cd /projects && (mvn clean -Dmaven.repo.local=/data/cache/.m2/repository package -Dmaven.test.skip=true) 1>>/proc/1/fd/1 2>>/proc/1/fd/2]: error while streaming command: command terminated with exit code 1
      
       •  Syncing files into the container  ...
      
 ✓  Syncing files into the container [1s]
       •  Building your application in container (command: defaultbuild)  ...
       ✗  Building your application in container (command: defaultbuild) [2s]
      Error occurred on Push - watch command was unable to push component: unable to exec command [/bin/sh -c cd /projects && (mvn clean -Dmaven.repo.local=/data/cache/.m2/repository package -Dmaven.test.skip=true) 1>>/proc/1/fd/1 2>>/proc/1/fd/2]: error while streaming command: command terminated with exit code 1
      
       •  Syncing files into the container  ...

[...]
```

```
[odo] [ERROR] No plugin found for prefix 'spring-boot' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories [local (/data/cache/.m2/repository), central (https://repo.maven.apache.org/maven2)] -> [Help 1]
  [odo] [ERROR] 
  [odo] [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
  [odo] [ERROR] Re-run Maven using the -X switch to enable full debug logging.
  [odo] [ERROR] 
  [odo] [ERROR] For more information about the errors and possible solutions, please read the following articles:
  [odo] [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/NoPluginFoundForPrefixException
```


## What versions of software are you using?

**Operating System:**

**Output of `odo version`:**

## How did you run odo exactly?


## Actual behavior


## Expected behavior


## Any logs, error output, etc?

--- a/tests/examples/source/devfiles/springboot/devfile-with-missing-projectType-and-language-metadata.yaml
+++ b/tests/examples/source/devfiles/springboot/devfile-with-missing-projectType-and-language-metadata.yaml
@@ -10,7 +10,7 @@ starterProjects:
 components:
   - name: tools
     container:
-      image: quay.io/eclipse/che-java11-maven:next
+      image: registry.access.redhat.com/ubi8/openjdk-11:latest
       memoryLimit: 768Mi
       command: ['tail']
       args: [ '-f', '/dev/null']
@@ -20,7 +20,7 @@ components:
       mountSources: true
   - name: runtime
     container:
-      image: quay.io/eclipse/che-java11-maven:next
+      image: registry.access.redhat.com/ubi8/openjdk-11:latest
       memoryLimit: 768Mi
       endpoints:
         - name: "8080-tcp"

--- a/tests/examples/source/devfiles/springboot/devfile-with-missing-projectType-metadata.yaml
+++ b/tests/examples/source/devfiles/springboot/devfile-with-missing-projectType-metadata.yaml
@@ -11,7 +11,7 @@ starterProjects:
 components:
   - name: tools
     container:
-      image: quay.io/eclipse/che-java11-maven:next
+      image: registry.access.redhat.com/ubi8/openjdk-11:latest
       memoryLimit: 768Mi
       command: ['tail']
       args: [ '-f', '/dev/null']
@@ -21,7 +21,7 @@ components:
       mountSources: true
   - name: runtime
     container:
-      image: quay.io/eclipse/che-java11-maven:next
+      image: registry.access.redhat.com/ubi8/openjdk-11:latest
       memoryLimit: 768Mi
       endpoints:
         - name: "8080-tcp"

--- a/tests/examples/source/devfiles/springboot/devfile-with-subDir.yaml
+++ b/tests/examples/source/devfiles/springboot/devfile-with-subDir.yaml
@@ -11,7 +11,7 @@ starterProjects:
 components:
   - name: tools
     container:
-      image: quay.io/eclipse/che-java11-maven:next
+      image: registry.access.redhat.com/ubi8/openjdk-11:latest
       memoryLimit: 768Mi
       command: ['tail']
       args: [ '-f', '/dev/null']
@@ -21,7 +21,7 @@ components:
           path: /data/cache/.m2
   - name: runtime
     container:
-      image: quay.io/eclipse/che-java11-maven:next
+      image: registry.access.redhat.com/ubi8/openjdk-11:latest
       memoryLimit: 768Mi
       endpoints:
         - name: "8080-tcp"

